src/components/Editor.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 import CodeMirror from 'react-codemirror';
-import _ from 'lodash';
+import isEqual from 'lodash/isEqual';
```

Comment:
Our current method, which is
```js
import _ from 'lodash';
```
will import _all_ of lodash (which is about 75kb), including the functions we don't use. This method instead imports only those functions we want.